recipe/meta.yaml

```diff
@@ -31,9 +32,10 @@ requirements:
     - six
 
 test:
-  imports:
-    - spiceypy
-    - spiceypy.utils
+  requires:
+    - pytest
+  commands:
+    - py.test --pyargs spiceypy
```

Comment:
Only one thing, use `pytest` instead of `py.test`!

Reply:
done, but not sure that will fix it...

Reply:
Yeah, it was not to fix it, but to use the "new" command